tests/rtl/mockServices/mockServer.js

```diff
@@ -0,0 +1,10 @@
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+
+import userMockServices from 'rtl/mockServices/userMockServices';
+
+const serverHandlers = [...userMockServices];
```

Comment:
```suggestion
const handlers = [...userMockServices];
```
**Trivial:** you can just call it handlers since you are already in mockServer file, and it is a litte bit redundant